server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -426,6 +429,14 @@ public int queryTimeoutSeconds() {
         return queryTimeoutSeconds;
     }
 
+    @JsonIgnore
```

Comment:
It's too customized. This class also needs to be oriented to other data sources. This way it will be too closely bound to OB.

Reply:
I don't think so. Think about the `clusterName`, `tenantName`, `sysTenantUsername`, and `sysTenantPassword` in `ConnectionConfig`, they are also OceanBase-specific fields. Additionally, the method `getDefaultSchema` is also bound to `DialectType`. 

On the other hand, I don't think that checking if the dialect is OceanBase within this `getInternalSessionInitScript` contradicts what you've said about this class being oriented to other data sources.

It's unlikely that you would use only one class to represent all data sources without introducing features specific to different types of data sources within this class.